docs/rules/no-only-tests.md

```diff
@@ -0,0 +1,49 @@
+# Disallow the test case property `only` (no-only-tests)
+
+The [`only` property](https://eslint.org/docs/developer-guide/unit-tests#running-individual-tests) can be used as of [ESLint 7.29](https://eslint.org/blog/2021/06/eslint-v7.29.0-released#highlights) for running individual rule test cases with less-noisy debugging. This feature should be only used in development, as it prevents all the tests from running. Mistakenly checking-in a test case with this property can cause CI tests to incorrectly pass.
+
+## Rule Details
+
+This rule flags a violation when a test case is using `only`. Note that this rule is not autofixable since automatically deleting the property would prevent developers from being able to use it during development.
```

Comment:
maybe we can add a suggestion to fix it?

Reply:
Added a suggestion to remove the the `only: true` property.